channels/rdpei/client/CMakeLists.txt

```diff
@@ -30,3 +30,9 @@ set(${MODULE_PREFIX}_LIBS
 include_directories(..)
 
 add_channel_client_library(${MODULE_PREFIX} ${MODULE_NAME} ${CHANNEL_NAME} TRUE "DVCPluginEntry")
+
+option(RDPEI_USE_CHANNEL_THREAD "use a thread to process RDPEI channel events" OFF)
```

Comment:
Since we already have `FreeRDP_SynchronousDynamicChannels` (and the thread-less version makes only sense in that scenario) I'd check for that setting instead of introducing another build option.

Reply:
done.